go.mod

```diff
@@ -4,20 +4,20 @@ go 1.17
 
 require (
 	github.com/ereslibre/kube-webhook-wrapper v0.0.1
-	github.com/go-logr/logr v0.4.0
-	github.com/google/go-cmp v0.5.6
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.16.0
+	github.com/go-logr/logr v1.2.3
+	github.com/google/go-cmp v0.5.7
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.17.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/common v0.26.0
-	go.opentelemetry.io/otel v0.20.0
-	go.opentelemetry.io/otel/exporters/otlp v0.20.0
-	go.opentelemetry.io/otel/metric v0.20.0
-	go.opentelemetry.io/otel/sdk/metric v0.20.0
-	k8s.io/api v0.22.1
-	k8s.io/apimachinery v0.22.1
-	k8s.io/client-go v0.22.1
-	sigs.k8s.io/controller-runtime v0.10.0
+	go.opentelemetry.io/otel v1.7.0
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric v0.30.0
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.30.0
+	go.opentelemetry.io/otel/metric v0.30.0
+	go.opentelemetry.io/otel/sdk/metric v0.30.0
+	k8s.io/api v0.23.5
+	k8s.io/apimachinery v0.23.5
+	k8s.io/client-go v0.23.5
+	sigs.k8s.io/controller-runtime v0.11.2
```

Comment:
As in the https://github.com/kubewarden/kubewarden-controller/pull/166#discussion_r862030584, @kubewarden/kubewarden-developers  , what are your thought about updating controller-runtime?

Reply:
Fine with me, maybe double check what kubebuilder is using nowadays...

But @ereslibre and @raulcabello are the real experts there

Reply:
fine with me too. We should check nothing breaks, but we should update at some point.

Reply:
Great! I'll  test this change with the e2e tests. If it passes, I'll merge it.

Reply:
kube-builder is also using the v0.11.2 version in [latest tag](https://github.com/kubernetes-sigs/kubebuilder/blob/v3.4.0/go.mod#L17)